test/unit/transactor/status_test.rb

```diff
@@ -249,5 +249,21 @@ def setup
 
       assert_equal @application.id,  doc.at('status application id').content
     end
+
+    test '#to_xml lists application keys section when list_app_keys ext is enabled' do
+      usage  = {:month => {@metric_id.to_s => 429}}
+      status = Transactor::Status.new(service_id: @service_id,
+                                      application: @application,
+                                      list_app_keys: 1,
+                                      values: usage,
+                                      oauth: true)
+
+      doc = Nokogiri::XML(status.to_xml)
+
+      keys = doc.at 'app_keys'
+      assert keys
+      assert_equal @application.id, keys['app']
+      assert_equal @service_id, keys['svc']
+    end
```

Comment:
Would be good to test that the keys are there.

Reply:
In this case there are no keys, but I've changed to test that... there are no keys :). I will create another test with keys.